### PR TITLE
iOS: os::last_os_error fallout

### PR DESCRIPTION
**Pull request by vhbit**

### CODE REVIEW

File: src/rust-installer [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 49cc7f6fef12bdd77a0f8b182d9a64c371cb17c8
+Subproject commit 60fd8abfcae50629a3fc664bd809238fed039617
```
Comment (alexcrichton) [inline src/rust-installer]: I think this was reverted by accident

Comment (vhbit) [inline src/rust-installer]: Oops, definitely.

**Comment by vhbit** [comment]
@alexcrichton fixed accidentally changed rust-installer, should be ready to go.

**Pull request merged**